billing.md

```diff
@@ -1922,6 +1923,13 @@ When this method is invoked, a new checkbox will be available to the customer th
 > **Warning**  
 > If you have already configured [automatic tax collection](#tax-configuration) in your application's service provider then this feature will be enabled automatically and there is no need to invoke the `collectTaxIds` method.
 
+<a name="guest-checkout"></a>
+## Guest Checkout
+
+You might want to use guest checkouts for users that are not registered. When this method is called Cashier automatically creates a guest checkout session for you and you can inject you items, session configuration as you are used to via `create`.
+
+    $checkout = Checkout::guest()->create($items, $sessionOptions);
```

Comment:
Maybe also add an example of how they can apply a discount here:

`->withDiscount()`

Reply:
Thank you, I added an example. I'll dive into the implementation this week and could provide some detailed PR for this docs.